tests/tensorflow/test_models/inception_resnet_v2.py

```diff
@@ -142,12 +144,21 @@ def inception_resnet_block(x, scale, block_type, block_idx, activation="relu"):
         mixed, backend.int_shape(x)[channel_axis], 1, activation=None, use_bias=True, name=block_name + "_conv"
     )
 
-    x = layers.Lambda(
-        lambda inputs, scale: inputs[0] + inputs[1] * scale,
-        output_shape=backend.int_shape(x)[1:],
-        arguments={"scale": scale},
-        name=block_name,
-    )([x, up])
+    x = CustomScaleLayer(scale)([x, up])
     if activation is not None:
         x = layers.Activation(activation, name=block_name + "_ac")(x)
     return x
+
+
+class CustomScaleLayer(keras_layers.Layer):
```

Comment:
Copied from keras InceptionResNetV2 implementation in order for graphs to be equal